src/commands/updates/browser.ts

```diff
@@ -84,7 +88,7 @@ async function writeUpdateFileToDisk(
   channel: string,
   updateObject: {
     updates: {
-      update: Record<string, any>
+      update: Record<string, string | undefined>
```

Comment:
That is because [this commit](https://github.com/pulse-browser/gluon/commit/b5f52144b48fd92d632fbd4b524122a09e3ea59e) fixed it, but I don’t want to create merge conflicts

Reply:
Fair